src/modules/sponsor/setEdition.ts

```diff
@@ -45,8 +48,7 @@ export default async (edition: string) => {
     const sponsorClientForSponsorAccount = new Sponsor(getIOContext(), IOClientOptions)
     await sponsorClientForSponsorAccount.setEdition(previousAccount, previousWorkspace, edition)
 
-    const workspaceNotice = previousWorkspace === 'master' ? '' : `in workspace ${chalk.blue(previousWorkspace)} `
-    log.info(`Successfully set edition ${workspaceNotice}of account ${chalk.blue(previousAccount)}.`)
+    log.info(`Successfully set edition${workspaceNotice} of account ${chalk.blue(previousAccount)}.`)
```

Comment:
`_nit_`: maybe should be this? 
```suggestion
    log.info(`Successfully set edition ${workspaceNotice} of account ${chalk.blue(previousAccount)}.`)
```